File: cfgs/java_cfgs_foo/MINIMUM_COST_FOR_ACQUIRING_ALL_COINS_WITH_K_EXTRA_COINS_ALLOWED_WITH_EVERY_COIN.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int coins_needed = ((int) (fun_1((1.0 * foo_1) / (foo_2 + 1))));15;
4;statement;int ans = 0;16;
5;statement;int i = 0;17;
6;conditional;foo_1 <= (foo_2 - 1);18;
7;statement;ans += foo_1[foo_2];19;
8;statement;i++;19;
9;statement;return foo_1;20;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;9;
6;8;
7;6;
8;10;
9;

File: cfgs/pythoncfgs/MINIMUM_COST_FOR_ACQUIRING_ALL_COINS_WITH_K_EXTRA_COINS_ALLOWED_WITH_EVERY_COIN.py
Nodes
1;statement;fun_1();10;
2;statement;foo_1 = fun_1(1.0 * foo_2 // (foo_3 + 1));11;
3;statement;foo_1 = 0;12;
4;conditional;fun_1(foo_1 - 1 + 1);13;
5;statement;return foo_1;15;
6;statement;foo_1 += foo_2[foo_3];14;
7;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;7;
5;4;
6;
